src/devTools.js

```diff
@@ -138,6 +143,7 @@ function liftReducer(reducer, initialState) {
         currentStateIndex++;
       }
       stagedActions = [...stagedActions, liftedAction.action];
+      timestamps = [...timestamps, Date.now()];
       break;
```

Comment:
Gotcha. For RESET, COMMIT, ROLLBACK, where the timestamp array is reset, do you think it's better to put the timestamp in their action creators as well? Since ```liftAction()``` is a de facto action creator so that would make it a consistent pattern.